app/Http/Controllers/UserController.php

```diff
@@ -79,6 +100,16 @@ public function update(Request $request, $id)
      */
     public function destroy($id)
     {
-        //
+        $user = User::findOrFail($id);
+
+        $user->delete();
+        alert()->warning(trans('trans.success'), trans('trans.suc_delete'));
+        return back();
     }
+
+    public function back(){
+        
+        return redirect()->route("user.index");
+    }
```

Comment:
```suggestion
    public function back(){
        return redirect()->route("user.index");
    }
```
Thừa dòng.
Với lại function này để làm gì vậy?

Reply:
cái back này để chạy nút cancel trong form edit với add để nó quay về trang index của model tương ứng thôi.

Reply:
vậy tại sao ko redirect trực tiếp luôn nhỉ? lại phải làm thêm một hàm nữa?